app/views/components/branded-header/README.md

```diff
@@ -0,0 +1,17 @@
+Use the MyHMCTS Branded Header component for any service or system not hosted on GOV.UK such as those for the judiciary or HMCTS staff.
```

Comment:
I think this needs expanding. I don't know what 'MyHMCTS' is and when I would use this instead of the normal header.